tests/SenderTest.php

```diff
@@ -234,6 +234,31 @@ public function testSendSticker()
         $this->assertCorrectRequestBody($request, json_encode($expectedBody));
     }
 
+    public function testErrorResponses()
+    {
+        $container = [];
+        $sender = $this->generateSender(
+            $this->generateHandlerStack(
+                $container,
+                $this->generateMockHandlerToResponseErrors()
+            )
+        );
+
+        $sender->sendSticker(
+                [self::TO_USER_ID_FOR_TEST],
+                'sticker_id',
```

Comment:
__[code_sniffer]__
* Multi-line function call not indented correctly; expected 12 spaces but found 16